lib/plottr_components/src/components/dashboard/account/proOnboarding/ProStep2.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'react-proptypes'
+import { t } from 'plottr_locales'
+import OnboardingStep from '../../../onboarding/OnboardingStep'
+import { StepBody, StepHeader } from '../../../onboarding/Step'
+import UnconnectedUploading from './Uploading'
+
+// is there a way to know that this step is done?
+
+const isPlottrCloudFile = (filePath) => filePath && filePath.startsWith('plottr://')
```

Comment:
We should probably think about abstracting this out at some point.
I think it's repeated about 4 times in the code base now :/